--- redist/goldendict.desktop	2010-12-04 00:12:46.000000000 +0300
+++ redist/goldendict.desktop.new	2011-08-30 22:59:09.000000000 +0400
@@ -3,8 +3,11 @@
 Terminal=false
 Categories=Office;Dictionary;Education;Qt;Applications
 Name=GoldenDict
+Name[ru]=Словарь Goldendict
 GenericName=Multiformat Dictionary
+GenericName[ru]=Многоформатный словарь-переводчик
 Comment=GoldenDict
+Comment[ru]=Многоформатный словарь-переводчик
 Encoding=UTF-8
 Icon=/usr/share/pixmaps/goldendict.png
 Exec=goldendict
